services/webnn/ort/graph_builder_ort.cc

```diff
@@ -590,6 +593,40 @@ void GraphBuilderOrt::AddConv2dOperation(const mojom::Conv2d& conv2d) {
   CHECK_STATUS(GetOrtGraphApi()->AddNode(graph_.get_ptr(), node.get_pptr()));
 }
 
+void GraphBuilderOrt::AddExpandOperation(const mojom::Expand& expand) {
+  const std::string node_name = GetNodeName(expand.label);
+  const std::string input_name = GetOperandName(expand.input_operand_id);
+  const std::string output_name = GetOperandName(expand.output_operand_id);
+
+  const OperandDescriptor& output_descriptor =
+      GetOperand(expand.output_operand_id).descriptor;
+  const std::vector<uint32_t>& output_shape = output_descriptor.shape();
+  // Shape is an operand with data type int64, not an attribute.
+  std::vector<uint32_t> shape_dims = {
+      base::checked_cast<uint32_t>(output_shape.size())};
+  std::vector<int64_t> shape_values;
+  base::ranges::transform(
+      output_shape, std::back_inserter(shape_values),
+      [](uint32_t dim) { return static_cast<int64_t>(dim); });
+  uint64_t shape_id = NewInitializerAsRawData(
```

Comment:
Out of curiosity: how do we know ORT requires the shape to be initialized as raw data? Adding a comment to explain?